deps/pathjoin.lua

```diff
@@ -9,12 +9,7 @@
 
 local getPrefix, splitPath, joinParts
 
-local isWindows
-if _G.jit then
-  isWindows = _G.jit.os == "Windows"
-else
-  isWindows = not not package.path:match("\\")
-end
+local isWindows = require("los").type() == "win32"
```

Comment:
This is part of the bigger issue of code replicates mentioned in #1166, but can you also apply this patch to luvipath.lua in luvi, or is it not worth the effort?

Reply:
A similar patch could be applied to luvi, but the isWindows check is one of the things that likely can't change; `los` isn't available for luvi because its a luvit library.